config/prod/prow/config_knative.yaml

```diff
@@ -83,6 +95,19 @@ presubmits:
         memory: 12Gi
       limits:
         memory: 16Gi
+  - custom-test: istio-stable-mesh-short
+    needs-monitor: true
+    always-run: false
+    optional: true
+    run-if-changed: ^third_party/istio-latest/net-istio.yaml
```

Comment:
shouldn't we run when istio-stable changes for this job?

Reply:
The path `^third_party/istio-latest/net-istio.yaml` is not net-istio repo's but for serving repo's. So only `istio-latest` is available https://github.com/knative/serving/tree/main/third_party/istio-latest :sweat_drops: